text/0110-z-pages.md

```diff
@@ -0,0 +1,31 @@
+# zPages: general direction (#110)
+
+Make zPages a standard OpenTelemetry component.
+
+## Motivation
+
+ZPages are in-process web pages that display collected data from the process they are attached to. They are used to provide in-process diagnostics without the need of any backend to examine traces or metrics. Various implementations of zPages are widely used in many environments. The standard extensible implementation of zPages in OpenTelemetry will benefit everybody.
+
+## Explanation
+
+This OTEP is a request to get a general approval for zPages development as an experimental feature [open-telemetry/opentelemetry-specification#62](https://github.com/open-telemetry/opentelemetry-specification/pull/632). See [opencensus.io/zpages](https://opencensus.io/zpages/) for the overview of zPages.
+
+## Internal details
+
+This is a request for a general direction approval, not specific details.
```

Comment:
I think this is insufficient detail to request an "approval". I would prefer to establish goals (dos and donts) and requirements. Also a discussion of pros and cons would be appropriate for an OTEP, considering that there is prior art and presumably prior learnings.

Reply:
I have added principles. Does it sound OK now? I really want to simplify the process of experimentation lightweight. As you pointed - there is a prior art and hopefully it will guide the experiment to the right direction.